pph_ecc/reedsolomon.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, threshold, secretdata=None):
 
       for byte in self.secretdata:
 		  
-		  byte = rsman.encode(byte)
+		  byte = rsman.encode_fast(byte) #use encode_fast which used faster algorithms and optimization tricks
```

Comment:
I would drop the comment. It is clear by the code itself that "fast" is faster :P